src/future/server/mod.rs

```diff
@@ -325,15 +331,15 @@ struct BindStream<S, St> {
 
 impl<S, St> fmt::Debug for BindStream<S, St>
     where S: fmt::Debug,
-          St: fmt::Debug,
+          St: fmt::Debug
 {
     fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
         const HANDLE: &'static &'static str = &"Handle { .. }";
         f.debug_struct("BindStream")
```

Comment:
Hmm, yes, that does seem odd. May be a bug? Unclear if it's better to break it though.